frontend/src/components/jobs/result/pluginReportTables.jsx

```diff
@@ -120,11 +159,26 @@ const tableProps = {
 
 export function AnalyzersReportTable({ job, refetch }) {
   console.debug("AnalyzersReportTable rendered");
+  const reports = job?.analyzer_reports;
+
+  const [analyzers, analyzersLoading] = usePluginConfigurationStore((state) => [
+    state.analyzers,
+    state.analyzersLoading,
+  ]);
+
+  reports.forEach((report, index) => {
+    analyzers.forEach((analyzer) => {
+      if (analyzer.name === report.name) {
+        reports[index].description = analyzer.description;
+      }
+    });
+  });
```

Comment:
This logic is used for each plugin, can we create a generic function to avoid code duplication ?